joomla/joomla-cms
# [com_newsfeeds] add searchtools to modal layout and solves bugs

#### Description

This PR adds searchtools to com_newsfeeds modal layout newsfeeds view.
The modal hasn't working properly (wasn't forcing the language in language association modal).

Also, in normal layout:
- correct bug: ordering the view for Association doesn't add the arrow icon (removed ordering in this column).
- improves responsive visual (mobile/tablet/desktop)
- solves other minor graphical issues.
- minor code style improvements
###### Before PR (modal)

![before-pr](https://cloud.githubusercontent.com/assets/9630530/13295688/f3ca9e0a-db21-11e5-962e-02fb190062ce.png)
###### After PR  (modal)

![after-pr](https://cloud.githubusercontent.com/assets/9630530/13295694/f8411b6c-db21-11e5-95d0-ad9b035d4b81.png)
#### How to test

Normal view:
1. Install latest staging and apply this patch
2. Go to Components -> News Feeds
3. Check if search, filters, ordering and pagination are working properly
4. Test on mobile and tablet (ex: reduce browser width)

Modal view:
1. Enable language associations.
2. Create some news feeds in several languages.
3. Edit one of those news feeds and go to "Associations" tab and click on "Select" on any language.
4. Check if search, filters, ordering and pagination are working properly in modal.
5. Test on mobile and tablet (ex: reduce browser width)
